win/rtpdump.vcxproj

```diff
@@ -94,12 +94,10 @@
     <ClCompile Include="../notify.c" />
     <ClCompile Include="../rd.c" />
     <ClCompile Include="../rtpdump.c" />
-    <ClCompile Include="../win\gettimeofday.c" />
```

Comment:
Please add the line bellow for the all four vcproj files to be linked successfully on Win. Other should be fine for the build.
<ClCompile Include="../compat-gettimeofday.c" />